tests/test_comparison_level_composition.py

```diff
@@ -5,69 +5,85 @@
 from splink.duckdb.duckdb_linker import DuckDBLinker
 
 
-def test_composition_internals():
-    composition_funs = {
-        "OR": cll.cl_or,
-        "AND": cll.cl_and,
-    }
+def test_misc_checks():
+    level = cll.not_(cll.null_level("first_name")).__dict__
+    assert level["_level_dict"].get("is_null_level") is None
```

Comment:
See above: Should be `is False` (or == False? not sure what ruff thinks), not `is None`

Reply:
I've changed this to be `False` as I think you're correct that it makes more sense for the user.